filament/src/details/Skybox.h

```diff
@@ -60,6 +57,7 @@ class FSkybox : public Skybox {
     FMaterialInstance* mSkyboxMaterialInstance = nullptr;
     utils::Entity mSkybox;
     FRenderableManager& mRenderableManager;
+    float mIntensity = 0.0f;
```

Comment:
I think the = 0.0f is ignored by the compiler because of the initializer list in the constructor.